dataScrape/src/pokemonTableUpdate.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class pokemonTableUpdate
```

Comment:
Classes should be capitalized:

PokemonTableUpdate

Reply:
Changed in bca46ad